components/dash-core-components/src/components/RangeSlider.react.js

```diff
@@ -19,6 +19,21 @@ export default class RangeSlider extends Component {
 }
 
 RangeSlider.propTypes = {
+    /**
```

Comment:
I think, calculating initial `ticks`, `step` and `value` properties are missing here if we want to use it as it's used int the format spec: https://gist.github.com/workaholicpanda/e54d762fd8ae6e755eebfbef40c42bec

It renders, but probably not the way we would expect, so it's not really usable if we omit all props other than `min` and `max`.

Reply:
I'm open to add that or copy things from your PR if it's okay. @akksi